### PR TITLE
ermbr-4182 Empty workflow so I can test a complete workflow from a branch

### PR DESCRIPTION
**Pull request by cesarmax22**
## Goal

Pushing an empty workflow, so once merged to master, I can test a complete workflow from a branch.

## Release Notes

**WHAT**:<br>
**WHY**:<br>

### CODE REVIEW

File: .github/workflows/swazzler-tests-subset.yml [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+on:
+  workflow_call:
+
+env:
+  ANDROID_BUILD_TOOLS_HOME: "/usr/local/lib/android/sdk/build-tools/32.0.0"
+
+jobs:
+  testGradle:
+    name: "Run Gradle Tests"
+#    runs-on: ubuntu-22.04-8cores
```
Comment (fractalwrench) [inline .github/workflows/swazzler-tests-subset.yml:12]: Can you delete any commented out code?

File: .github/workflows/cesar-test.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+name: "Test swazzler-tests"
+
+on:
```
Comment (fractalwrench) [inline .github/workflows/cesar-test.yml:3]: This should specify the job should run on every pull request to master

```

  pull_request:
    branches: [ master ]
```

**Review by fractalwrench — COMMENTED** [review]
This will be cool! Do you have an example of the job running against master?

**Review by bidetofevil — APPROVED** [review]
LGTM.

**Pull request merged**